app/models/user.rb

```diff
@@ -14,6 +14,12 @@ class User < ApplicationRecord
 
   belongs_to :organization, optional: true
 
+  has_many :roles
+  include RoleScopedRegions
+  has_many :managed_organizations, -> { Role.super_admin_or_admin }, through: :roles, source: :resource, source_type: "Organization"
+  has_many :managed_facility_groups, -> { Role.super_admin_or_admin }, through: :roles, source: :resource, source_type: "FacilityGroup"
+  has_many :managed_facilities, through: :managed_facility_groups, source: :facilities
```

Comment:
I initially tried to get things working w/ these has_manys and AR merges, but it wasn't working out.  Not sure we would even want to keep these assocations honestly, probably better to have the one interface in RoleScopedRegions.